presto-docs/src/main/sphinx/release/release-0.225.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.225
+=============
+
+General Changes
+_______________
+* Fix the issue that fragment's output size was under-recorded for cases where input data is dictionary or run-length encoded, or the rows are replicated. (:issue:`11770`)
+* Fix leak in operator peak memory computations. (:issue:`13210`)
+* Allow overriding session zone sent by clients. (:issue:`13140`)
+* Improve memory tracking for spatial query indexing and avoid worker node crash due to out of memory. (:pr:`13251`)
+* Throw an exception when a spatial partition is made with 0 rows.
+* Add Flatbush RTree to replace JTS STR RTree for spatial query.
+* Add a simple histogram, and a histogram with breakdown of weights within each bin, which will be used by classification metrics and differential entropy functions.
+* Add `TableStatistics` in `QueryInputMetadata` for Join Query. (:pr:`12808`)
+* Add configuration property ``experimental.max-total-running-task-count`` and ``experimental.max-query-running-task-count``
+  to limit number of running tasks for all queries and a single query, respectively. Query will be killed only if both conditions are violated. (:pr:`13228`)
+
+JDBC Changes
+____________
+* Match schema and table names case insensitively. This behavior can be enabled by setting the ``case-insensitive-name-matching`` catalog configuration option to true.
+
+Web UI Changes
+______________
+* Display tasks per stage on the query page.
+
+Elasticsearch Connector Changes
+_______________________________
+* Fix nested fields in Elasticsearch Connector. (:issue:`12948`)
+
+Hive Changes
+____________
+* Use JSON key file to access Google Cloud Storage ``hive.gcs.json-key-file-path=/path/to/gcs_keyfile.json``.
+* Use client-provided OAuth token to access Google Cloud Storage ``hive.gcs.use-access-token=false``.
```

Comment:
Add ability to use a client-provided OAuth token to access Google Cloud Storage.  This can be configured using the configuration property ``hive.gcs.use-access-token=false``.